_includes/v22.2/backups/feature-availability-note.md

```diff
@@ -0,0 +1 @@
+This feature is available in **{{ site.data.products.dedicated }}**, **{{ site.data.products.serverless }}**, and **{{ site.data.products.core }}**.
```

Comment:
Hmm, I'm actually torn here. It is a little hard to know which this page applies to, until the more extensive left-nav CC vs SH changes are implemented (that Mike has been talking about), so I don't mind a little bit of explicitness. That being said, I would definitely prefer for applicability to be entirely determinable by left-nav when we get to that stage.

What do you think about retaining this explicit mention for now, until Mike's re-org proposal is completed?

Or: If you don't like it right up top as the first text on-page, I think a `## Supported Products` section or similar would also be fine, perhaps as a section of its own just before `## Use Key Management Service` on [this page](https://deploy-preview-16316--cockroachdb-docs.netlify.app/docs/dev/take-and-restore-encrypted-backups.html).

I really could go either way on this for the interim. Up to you.

Reply:
I really like the Supported products idea and so I have added this in. After discussing with Liv yesterday, it was clear that customers are confused by which product can do what. And so until our IA in general directs users this way, I think this heading provides the clarity needed.